conanfile_rtc.py

```diff
@@ -0,0 +1,23 @@
+from conans import ConanFile
+
+
+class SPDLogConan(ConanFile):
+    name = "spdlog"
+    version = "1.9.2"
+    url = "https://github.com/Esri/spdlog/tree/runtimecore"
+    license = "https://github.com/Esri/spdlog/blob/runtimecore/LICENSE"
+    description = "Very fast, header-only/compiled, C++ logging library."
```

Comment:
`Fast C++ logging library.` is actually what is on the about page of the library. I usually just copy paste that to stay in line with the upstream project.

Reply:
I copied the first line of the [readme](https://github.com/Esri/spdlog#spdlog)

Reply:
Ahh my bad, they usually keep the About and README the same.